openkfw/TruBudget
# #1833-fixed tables bottom margin

### Checklist




### Description
* added bottom margin for several tables

### How to test

1. log in to the app
2. go to user page and see that table has margin at the bottom
3. at the main page select a project that has a lot of subprojects and see that table has a bottom margin
4. select subproject and see that table has a bottom margin




Closes #1833
